src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,50 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Predicate;
+
+public class Lotto {
+    private List<Integer> lotto = new ArrayList<>();
+    private int correctNumberCount = 0;
+    private int prizeMoney = 0;
+
+    public Lotto() {
+        generateLotto(new LottoNumberAuto());
+    }
+
+    public void generateLotto(LottoNumber lottoNumber) {
+        this.lotto = lottoNumber.generateLottoNumber();
+    }
+
+    public List<Integer> getLotto() {
+        return Collections.unmodifiableList(lotto);
+    }
+
+    public int getPrizeMoney() {
+        return prizeMoney;
+    }
+
+
+    public int checkLottoNumbers(List<Integer> answerNumbers) {
+        correctNumberCount = (int) lotto.stream()
+                .filter(number -> answerNumbers.stream().anyMatch(Predicate.isEqual(number))).count();
```

Comment:
```suggestion
        matchCount = (int) numbers.stream()
            .filter(answerNumbers::contains)
            .count();
```

- stream을 한번 더 여는 것보단 contains로 같은 기능을 수행할 수 있지 않을까요
- Method Reference를 적극적으로 사용하는 것은 어떨까요

Reply:
Method Reference 에 대해서 공부해야봐야겠네요 감사합니다!!